packages/bpk-component-breakpoint/src/BpkBreakpoint.tsx

```diff
@@ -63,7 +66,10 @@ const BpkBreakpoint = ({
       );
     }
 
-    return <MediaQuery query={query}>{children}</MediaQuery>;
+    if (typeof children === 'function') {
+      return children(matches) as ReactElement;
+    }
+    return matches ? (children as ReactElement) : null;
```

Comment:
We're now doing a wee bit of extra logic here to accomodate the 2 different children types we can receive. This was copied directly from react-responsive: https://github.com/yocontra/react-responsive/blob/121c86ddcbcd2f5523dc364393be3605c90d0a0f/src/Component.ts#L26-L29